### DISCUSSION TITLE
Issue with Missing Customer Address Information

### DISCUSSION THREAD

**Discussion by Fayvorite008** [etsy/open-api#1339] [Q&A]
Could anyone kindly tell me if there are specific regions or conditions under which address information may not be provided through the API? Our clients from Thai says not for his shop but I'm not sure.  

The API is able to retrieve three orders from the customer, but due to the absence of address information (such as country, address line 1, address line 2, and city), we are unable to create the orders in our system. The customer’s shop is located in the U.S., but the person’s IP is from Thailand. What should we do in this case? THX

![image](https://github.com/user-attachments/assets/762c9536-b8a4-4be4-b9ea-2e9430198407)

**Comment by seandmccarthy** [comment]
They're not providing access for anyone other than their two announced shipping partners. See

https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2024-10-21

https://help.etsy.com/hc/en-us/articles/26654295371031-How-to-Use-a-Third-Party-Provider-to-Ship-Your-Order?segment=selling

https://www.ecommercebytes.com/2024/10/19/etsy-reportedly-ignores-pirate-ship-as-it-restricts-shipping-services/

**Comment by Clarry100** [comment]
I'm actually working on this same endpoint at the moment and the live data returned from etsy contains ALL of the fields you show as missing. 

Actually the endpoint we are using is getShopReceipts. [NOTE the -s- on the end.] 
Which gets all receipts according to requested filters.

Not too sure if/not it makes any difference but we,  and our etsy store, are located in Australia.

**Reply by seandmccarthy** [reply]
It affects stores in US, CA, and Turkey only. Until they make exclusive shipping deals in other countries...

**Comment by beeirl** [comment]
Have you been able to get any clarification from Etsy? In the release notes, they keep mentioning that "approved categories" will still be granted access. I’d assume that any non-shipping-related category falls under that, but it’d be great to get an official statement to confirm. I don't know why this has to be so tedious...
